app/src/main/java/app/revenge/manager/installer/step/download/DownloadLibsStep.kt

```diff
@@ -23,7 +23,7 @@ class DownloadLibsStep(
 
     override val nameRes = R.string.step_dl_lib
 
-    override val url: String = "$baseUrl/tracker/download/$version/config.$arch"
+    override val url: String = "/tracker/download/$version/config.$arch"
```

Comment:
Should be renamed to "path"

Reply:
I thought about this and initially did this, but not all steps use mirrors which mean they need to be a full url